### PR TITLE
Suite path fix

### LINKED ISSUE #191: Hierarchy issue for module_path on windows OS
i have **test_fix1.py** in **tests** folder,  after enable Hierarchy.

the hier part is **:\test_fix1.py** on windows OS, but **test_fix.py** on Mac OS.

print logs in _add_item_hier_parts_other function as below:

                    print('======modulepath=======')
                    print(rp_name)
                    print(part.fspath.basename)
                    print(item.fspath)
                    print(module_path)
                    rp_name = module_path if rp_name else module_path[1:]
                    print(rp_name )

output as below:

test_fix1.py
C:\Code\apiproject\tests\test_fix1.py
C:\test_fix1.py
:\test_fix1.py

seems the module_path[1:] is not correct on windows OS.

### PR DESCRIPTION
**Pull request by HardNorth**
Fixes #191

### CODE REVIEW

File: tests/integration/test_suite_hierarchy.py [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+        {'name': second_suite_name, 'item_type': 'SUITE',
+         'parent_item_id': lambda x: x is None},
+        {'name': test_name, 'item_type': 'STEP',
+         'parent_item_id': lambda x: x.startswith(second_suite_name)}
+    ]
+
+    verify_start_item_parameters(mock_client, expected_items)
+
+
+TEST_VARIABLES = \
```
Comment (iivanou) [inline tests/integration/test_suite_hierarchy.py:167]: Can you put these global variables below into a separate file? It's hard to concentrate on tests looking through the module. Thanks

File: tests/integration/test_suite_hierarchy.py [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+         'parent_item_id': lambda x: x is None},
+        {'name': 'Tests', 'item_type': 'SUITE',
+         'parent_item_id': lambda x: x.startswith(
+             'examples\\test_in_class_parameterized.py')},
+        {'name': 'test_in_class_parameterized[param]', 'item_type': 'STEP',
+         'parent_item_id': lambda x: x.startswith('Tests')}
+    ]
+]
+
+TEST_PARAMETERS = [(v, TEST_EXPECTED_ITEMS[i]) for i, v in
```
Comment (iivanou) [inline tests/integration/test_suite_hierarchy.py:251]: Can you please rename single character variables to something that may give an idea of their content?

File: README.rst [hunk 3]
```diff
@@ -113,7 +113,9 @@ The following parameters are optional:
...
 - :code:`rp_issue_id_marks = True` - Enables adding marks for issue ids (e.g. "issue:123456")
 - :code:`rp_verify_ssl = True` - Verify SSL when connecting to the server
-- :code:`rp_display_suite_test_file = True` In case of True, include the suite's relative file path in the launch name as a convention of "<RELATIVE_FILE_PATH>::<SUITE_NAME>". In case of False, set the launch name to be the suite name only - this flag is relevant only when "rp_hierarchy_module" flag is set to False
+- :code:`rp_display_suite_test_file = True` - In case of True, include the suite's relative file path in the launch name as a convention of "<RELATIVE_FILE_PATH>::<SUITE_NAME>". In case of False, set the launch name to be the suite name only - this flag is relevant only when "rp_hierarchy_module" flag is set to False
+- :code:`rp_mode = DEFAULT` - DEBUG or DEFAULT launch mode. DEBUG launches are displayed in a separate tab and not visible to anyone except owner
+- :code:`rp_hierarchy_dir_path_separator` - Path separator to display directories in test hierarchy. In case of empty value current system path separator will be used
```
Comment (iivanou) [inline README.rst:118]: I think this option is excessive. We can dynamically determine system path separator in the code.

Comment (HardNorth) [reply README.rst:118]: We can, but original issue says that test names are different for MacOS and Windows. And I agree with the case that user can run tests on different systems and wants to see the same hierarchy for both systems. For those users I introduced this variable.

Comment (iivanou) [reply README.rst:118]: The hierarchy will be the same if we apply the correct path separator to the test path parts. If you apply a non-system path separator through the rp_hierarchy_dir_path_separator option, it won't solve the problem.

**Comment by HardNorth** [comment]
Agent logic simplification moved into separate issue: #286
But simplification should be done with a new major release. Right now we are fixing an issue in 5.0.10.

**Review by iivanou — APPROVED** [review]
Looks good.

**Pull request merged**